blog/2024-06-13/index.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: linter-restricting-automatic-merge
+title: "Linter breaking change: restricting automatic merge"
+authors: [barthalion]
+tags: [linter]
+---
+
+Starting next week, June 17th, the linter will no longer allow enabling
+automatic merge for pull requests created by flatpak-external-data-checker,
+unless the app uses an extra-data source or has been added to the exceptions list.
+
+The verified apps utilizing `automerge-flathubbot-prs` before May 28th have been
+grandfathered and no explicit action is required from the maintainers. If your app
+does not meet these criteria, you can still request an exception by creating a
```

Comment:
This should link to https://docs.flathub.org/docs/for-app-authors/linter/#exceptions and https://docs.flathub.org/docs/for-app-authors/linter/#flathub-json-automerge-enabled